### DISCUSSION TITLE
Github Action Unexpected input(s)

### DISCUSSION THREAD

**Discussion by sigfrid** [community/community#24962] [Actions]
<p>I’m working on a simple Action</p>
<pre><code class="lang-auto"> steps:
    - uses: actions/checkout@v1
    - name: Publish to GitHub Packages
      uses: sigfrid/github-container-registry-action@main
      with:
        pat: ${{ secrets.CR_PAT }}
</code></pre>
<p>And when I run it, I get the following warning</p>
<blockquote>
<p>Unexpected input(s) ‘pat’, valid inputs are [‘entryPoint’, ‘args’]</p>
</blockquote>
<p>But looking at <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith" rel="noopener nofollow ugc">the documentation</a>, everything seems correct.</p>
<p>Am I missing anything here?</p>
<p>Thanks,<br>
Sig</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Your repository doesn’t contain any metadata <code>action.yml</code> file. You need to define the expected inputs in there. See:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file" target="_blank" rel="noopener">Creating a Docker container action - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file" target="_blank" rel="noopener">//docs.github.com/en/actions/creating-actions/creating-a-docker-container-action</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by sigfrid** [comment]
<p>Thanks for the reply.</p>
